File: j2me/src/GoVoyage/Entities/Vol.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package GoVoyage.Entities;

/**
 *
 * @author SASSOU
 */
public class Vol {
        int id;
    String dateDepart;
    String dateArrivee;

   


    public Vol( String dateDepart, String dateArrivee, String heureDepart, String heureArrivee, String nbrePassagers, String companie, String prixBillet, String classeBillet, String aereportDepart, String aeroportArrivee) {
        //this.id = id;
        this.dateDepart = dateDepart;
        this.dateArrivee = dateArrivee;
        this.heureDepart = heureDepart;
        this.heureArrivee = heureArrivee;
        this.nbrePassagers = nbrePassagers;
        this.companie = companie;
        this.prixBillet = prixBillet;
        this.classeBillet = classeBillet;
        this.aereportDepart = aereportDepart;
        this.aeroportArrivee = aeroportArrivee;
    }
    
    String heureDepart;
    String heureArrivee;
    String nbrePassagers;

    public Vol() {
        
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDateDepart() {
        return dateDepart;
    }

    public void setDateDepart(String dateDepart) {
        this.dateDepart = dateDepart;
    }

    public String getDateArrivee() {
        return dateArrivee;
    }

    public void setDateArrivee(String dateArrivee) {
        this.dateArrivee = dateArrivee;
    }

    public String getHeureDepart() {
        return heureDepart;
    }

    public void setHeureDepart(String heureDepart) {
        this.heureDepart = heureDepart;
    }

    public String getHeureArrivee() {
        return heureArrivee;
    }

    public void setHeureArrivee(String heureArrivee) {
        this.heureArrivee = heureArrivee;
    }

    public String getNbrePassagers() {
        return nbrePassagers;
    }

    public void setNbrePassagers(String nbrePassagers) {
        this.nbrePassagers = nbrePassagers;
    }

    public String getCompanie() {
        return companie;
    }

    public void setCompanie(String companie) {
        this.companie = companie;
    }

    public String getPrixBillet() {
        return prixBillet;
    }

    public void setPrixBillet(String prixBillet) {
        this.prixBillet = prixBillet;
    }

    public String getClasseBillet() {
        return classeBillet;
    }

    public void setClasseBillet(String classeBillet) {
        this.classeBillet = classeBillet;
    }

    public String getAereportDepart() {
        return aereportDepart;
    }

    public void setArroportDepart(String aereportDepart) {
        this.aereportDepart = aereportDepart;
    }

    public String getAeroportArrivee() {
        return aeroportArrivee;
    }

    public void setAeroportArrivee(String aeroportArrivee) {
        this.aeroportArrivee = aeroportArrivee;
    }
    String companie;
    String prixBillet;
    String classeBillet;
    String aereportDepart;
    String aeroportArrivee;
    
    
}

File: symfony/src/Pi/GeneralBundle/Entity/Vol.php
<?php

namespace Pi\GeneralBundle\Entity;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Vol {
   
    /**
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ORM\Id
     * @ORM\Column(type="integer")  
     */
    private $id;  
     /**
     * @ORM\Column(type="string",length=512)
     */ 
    private $date_depart;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $date_arrivee;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $heure_depart;
     /**
     * @ORM\Column(type="string",length=512)
     */
    private $heure_arrivee;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $nbr_passager;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $companie_aerienne;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $prix_billet;
    /**
     * @ORM\Column(type="string",length=512)
     */
    private $classe_billet;
   
        /**
     * @ORM\Column(type="string",length=512)
     */
    private $aeroportDepart;
        /**
     * @ORM\Column(type="string",length=512)
     */
    private $aeroportArrivee;
    function getAeroportDepart() {
        return $this->aeroportDepart;
    }

    function getAeroportArrivee() {
        return $this->aeroportArrivee;
    }

    function setAeroportDepart($aeroportDepart) {
        $this->aeroportDepart = $aeroportDepart;
    }

    function setAeroportArrivee($aeroportArrivee) {
        $this->aeroportArrivee = $aeroportArrivee;
    }

        
    function getId() {
        return $this->id;
    }

    function getDate_depart() {
        return $this->date_depart;
    }

    function getDate_arrivee() {
        return $this->date_arrivee;
    }

    function getHeure_depart() {
        return $this->heure_depart;
    }

    function getHeure_arrivee() {
        return $this->heure_arrivee;
    }

    function getNbr_passager() {
        return $this->nbr_passager;
    }

    function getCompanie_aerienne() {
        return $this->companie_aerienne;
    }

    function getPrix_billet() {
        return $this->prix_billet;
    }

    function getClasse_billet() {
        return $this->classe_billet;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setDate_depart($date_depart) {
        $this->date_depart = $date_depart;
    }

    function setDate_arrivee($date_arrivee) {
        $this->date_arrivee = $date_arrivee;
    }

    function setHeure_depart($heure_depart) {
        $this->heure_depart = $heure_depart;
    }

    function setHeure_arrivee($heure_arrivee) {
        $this->heure_arrivee = $heure_arrivee;
    }

    function setNbr_passager($nbr_passager) {
        $this->nbr_passager = $nbr_passager;
    }

    function setCompanie_aerienne($companie_aerienne) {
        $this->companie_aerienne = $companie_aerienne;
    }

    function setPrix_billet($prix_billet) {
        $this->prix_billet = $prix_billet;
    }

    function setClasse_billet($classe_billet) {
        $this->classe_billet = $classe_billet;
    }


}